src/apps/front-office/account/pages/RestPassword/RestPassword.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { trans } from "@mongez/localization";
+import { Form } from "@mongez/react-form";
+import Helmet from "@mongez/react-helmet";
+import PasswordInput from "src/apps/front-office/design-system/components/form/PasswordInput";
+import { Button } from "src/apps/front-office/design-system/components/ui/button";
+import Breadcrumb from "src/apps/front-office/design-system/layouts/Breadcrumb";
+import URLS from "src/apps/front-office/utils/urls";
+
+export default function RestPassword() {
+  return (
+    <section className="w-full text-secondary">
+      <Helmet title={trans("about-us.title")} />
```

Comment:
Helmet should be declared at the very beginning of the returned jsx, also make sure it has the correct translation key